### DISCUSSION TITLE
Can you implement a script?

### DISCUSSION THREAD

**Discussion by Alone0316** [ROM-builders/temporary#14523] [Ideas]
https://github.com/ROM-builders/temporary/discussions/14517
I have discussed about it here, can you check and implement it? it will help everyone who is facing that problem

**Comment by Apon77** [comment]
how to do it?

**Reply by Alone0316** [reply]
> how to do it?

It should be similar to the sync script in which we have fixed the sync problem for various situations like cannot checkout, rev list --head etc etc, i could try making a script 🤷🏻‍♀️🤷🏻‍♀️

**Reply by Apon77** [reply]
Try then. Make sure it only uses that script when necessary. I mean it should only run that when it identifies the problem first.. Without identifying the problem it should not run. It should only run when script is confirm that it needs to be run.

**Reply by Alone0316** [reply]
https://github.com/ROM-builders/temporary/pull/14539  
can you merge it? i m not sure if it will work or not, it needs to be tested
